.github/workflows/jest-balance.yml

```diff
@@ -0,0 +1,42 @@
+name: jest balancer
+on: [workflow_dispatch]
```

Comment:
Could/should we make this a cron? 

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
Great idea. What do you think would be a good schedule? Maybe 1x per week and maybe on like a Sunday so we dont spam the repo mid week?

Reply:
Yeah once a week sounds good (although might be a bit noisy?)

Reply:
lets do once every two weeks, should be good enough